TokisanGames/Terrain3D
# Fix Instancer working w/ vertex spacing

Superseded by #523 
Fixes #479 

To test at spacing <1, 1, >1
* painting, w/ slope and ALT slope
* erasing, w/slope and ALT slope
* Saving & reloading
* Removing regions where instances were created at all scales to ensure the instances are attached to the proper region
* Sculpting underneath instances

Key reminder: ALT to inverse slope. CTRL+SHIFT to remove all instances.

The only thing I've seen is if you remove a region when vertex_spacing != 0, the instances in an adjacent region transform slightly. They aren't removed nor moved, but they seem to grow, shrink, or shear slightly. It isn't noticable at vertex_spacing 1. It may be due to the way we're scaling the MMI laterally only.

Remaining issues from first commit:
* update_transforms < 1 doesn't work
* Vertex spacing transforms the MMI!! And working on fixing this broke everything else.
